ccmbioinfo/lfs
# CARDS-2333: Automatically check in forms that haven't been modified in more than 30 minutes

To test **on the CARDS-2333-test branch**:

- build and start in prems mode
- create a new form, leave it in edit mode, check (in a different tab) that after two minutes the form is checked in
- save & view, edit again, wait one minute, make sure the form isn't checked in, make a change and save&continue editing, wait one minute, make sure the form isn't checked in, wait another minute, make sure the form is checked in
- generate a Visit Information form, wait 2 minutes, make sure the Survey Status form is checked in; fill the survey as a patient, submit, make sure the Survey Status form was updated to include the Survey Submited date.
